tests/test_crewai_adapter.py

```diff
@@ -364,3 +364,133 @@ def test_none_values_filtered_from_kwargs(mcp_server_that_rejects_none_script):
+    return dedent(
+        """
+        import mcp.types as types
+        from mcp.server.lowlevel import Server
+        from mcp.server.stdio import stdio_server
+        import anyio
+
+        app = Server("single-content-server")
+
+        @app.call_tool()
+        async def call_tool(name: str, arguments: dict | None) -> list[types.TextContent]:
+            if name != "single_content_tool":
+                raise ValueError(f"Unknown tool: {name}")
+            
+            # Return single content response
+            return [types.TextContent(
+                type="text", 
+                text="Single response"
+            )]
+
+        @app.list_tools()
+        async def list_tools() -> list[types.Tool]:
+            return [
+                types.Tool(
+                    name="single_content_tool",
+                    description="A tool that returns single content",
+                    inputSchema={
+                        "type": "object",
+                        "properties": {},
+                    },
+                )
+            ]
+
+        async def arun():
+            async with stdio_server() as streams:
+                await app.run(
+                    streams[0], streams[1], app.create_initialization_options()
+                )
+
+        anyio.run(arun)
+        """
+    )
+
+
+@pytest.fixture
+def multiple_content_server_script():
+    return dedent(
+        """
+        import mcp.types as types
+        from mcp.server.lowlevel import Server
+        from mcp.server.stdio import stdio_server
+        import anyio
+
+        app = Server("multiple-content-server")
+
+        @app.call_tool()
+        async def call_tool(name: str, arguments: dict | None) -> list[types.TextContent]:
+            if name != "multiple_content_tool":
+                raise ValueError(f"Unknown tool: {name}")
+            
+            # Return multiple content responses
+            return [
+                types.TextContent(type="text", text="First response"),
+                types.TextContent(type="text", text="Second response"),
+                types.TextContent(type="text", text="Third response")
+            ]
+
+        @app.list_tools()
+        async def list_tools() -> list[types.Tool]:
+            return [
+                types.Tool(
+                    name="multiple_content_tool",
+                    description="A tool that returns multiple content items",
+                    inputSchema={
+                        "type": "object",
+                        "properties": {},
+                    },
+                )
+            ]
+
+        async def arun():
+            async with stdio_server() as streams:
+                await app.run(
+                    streams[0], streams[1], app.create_initialization_options()
+                )
+
+        anyio.run(arun)
+        """
+    )
+
+
+def test_single_content_response_handling(single_content_server_script):
```

Comment:
I think this is already tested above so we don't need the single content test

Reply:
👍